src/SwedbankPay.Sdk.Infrastructure/ProblemResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+
+namespace SwedbankPay.Sdk
+{
+    internal class ProblemResponse : IProblemResponse

```

Comment:
We have a few classes and interfaces named `Response` that does not have a `Request` counterpart. What would be the consequence if we removed `Response` from the name of all of them?

```suggestion
    internal class Problem : IProblem
```

Reply:
Unsure, but it makes it easy to know what the class is used for when it has the `Response` or `Request` postfix.
Do you want me to remove `Response` postfix from places I can?

Reply:
Yes, let's try and see.

Reply:
I've updated `Problem` but I have not found any other good candidates for this after a short perusal.

Reply:
The `Response` from `Problem` and all related classes should be gone. I tried saying on Slack, but you seem to have missed it.